alvr/common/src/inputs.rs

```diff
@@ -261,6 +262,30 @@ pub static CONTROLLER_PROFILE_INFO: Lazy<HashMap<u64, InteractionProfileInfo>> =
                 .collect(),
             },
         ),
+        (
+            *PICO_G3_CONTROLLER_PROFILE_ID,
+            InteractionProfileInfo {
+                path: PICO_G3_CONTROLLER_PROFILE_PATH,
+                button_set: [
+                    *LEFT_MENU_CLICK_ID,
+                    *LEFT_TRIGGER_CLICK_ID,
+                    *LEFT_TRIGGER_VALUE_ID,
+                    *LEFT_THUMBSTICK_Y_ID,
+                    *LEFT_THUMBSTICK_X_ID,
+                    *LEFT_THUMBSTICK_CLICK_ID,
+                    *LEFT_THUMBSTICK_TOUCH_ID,
```

Comment:
The G3 controller has a touchpad, not a thumbstick. use the exact inputs available on the controller, the server will automatically map it to the emulated controllers, mapping it to a thumbstick if necessary.

Reply:
I did it because OpenXR documentation talks about thumbstick only, but it seems that runtime supports trackpad and Unity OpenXR Integration SDK uses it.